scripts/vagrantCloudBasedContainers.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh -ex
+
+UNIVERSAL_IMAGE=vagrant-universal
+
+COUCHDB_IMAGE=couchdb
+COUCHDB_PORT=5984
+
+DATALOADER_IMAGE="gpii/preferences-dataloader"
+DATALOADER_CLEAR_INDEX=1
+DATALOADER_COUCHDB_URL="http://couchdb:${COUCHDB_PORT}/preferences"
+
+PREFERENCES_CONFIG="gpii.config.preferencesServer.standalone.production"
+PREFERENCES_DATASOURCE_URL="http://couchdb:${COUCHDB_PORT}/preferences/%userToken"
+PREFERENCES_DIR="/home/vagrant/sync/universal/testData/preferences"
+PREFERENCES_PORT=9081
+
+FLOWMANAGER_CONFIG="gpii.config.cloudBased.flowManager.production"
+FLOWMANAGER_PREFERENCES_URL="http://preferences:${PREFERENCES_PORT}/preferences/%userToken"
+FLOWMANAGER_PORT=9082
+
+
+# Remove old containers (exit code is ignored)
+docker rm -f couchdb 2>/dev/null || true
+docker rm -f preferences 2>/dev/null || true
+docker rm -f flowmanager 2>/dev/null || true
+
+# Remove old image (exit code is ignored)
+docker rmi -f $UNIVERSAL_IMAGE 2>/dev/null || true
+
+# Build image
+docker build -t $UNIVERSAL_IMAGE .
+
+# Start containers and load preferences data
+docker run -d -p $COUCHDB_PORT:$COUCHDB_PORT --name couchdb $COUCHDB_IMAGE
+sleep 5
```

Comment:
This is probably fine for now. I guess an alternative would be to wait for localhost:$COUCHDB_PORT to be open, or even for localhost:$COUCHDB_PORT to return 200.